modules/37_industry/subsectors/equations.gms

```diff
@@ -231,15 +231,16 @@ q37_demFeFeedstockChemIndst(t,regi,entyFe,emiMkt)$(
     + pm_cesdata(t,regi,in,"offset_quantity")
     )
   * p37_chemicals_feedstock_share(t,regi)
-  )
+  )$( entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt) )
 ;
 
 *' Feedstocks flow has to be lower than total energy flow into the industry
 q37_feedstocksLimit(t,regi,entySe,entyFe,emiMkt)$(
-                                             sefe(entySe,entyFe)
-                                         AND sector2emiMkt("indst",emiMkt)
-                                         AND entyFe2Sector(entyFe,"indst")
-                                         AND entyFeCC37(entyFe)            ) ..
+                         sefe(entySe,entyFe)
+                     AND sector2emiMkt("indst",emiMkt)
+                     AND entyFe2Sector(entyFe,"indst")
+                     AND entyFeCC37(entyFe)
+                     AND entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt) ) ..
```

Comment:
I think you can delete the three lines above that, they should all be covered by the set with the endless name.